contracts/modules/capital/MCR.sol

```diff
@@ -96,13 +92,13 @@ contract MCR is IMCR, MasterAware {
     // IPriceFeedOracle priceFeed = pool.priceFeedOracle();
     // address daiAddress = priceFeed.daiAddress();
 
-    // uint ethAmount = qd.getTotalSumAssured("ETH").mul(1e18);
-    // uint daiAmount = qd.getTotalSumAssured("DAI").mul(1e18);
+    // uint ethAmount = qd.getTotalSumAssured("ETH") * 1e18;
```

Comment:
why do we have this commented out instead of removed?

Reply:
Removed ✅